### DISCUSSION TITLE
Any recommendations for cloud providers?

### DISCUSSION THREAD

**Discussion by davidtweedle** [mlcommons/algorithmic-efficiency#718] [Q&A]
Hi,
I am wondering if anyone has any suggestions for a cloud providers to use. I have been trying for the past few days to use Google Cloud Platform (an n1 instance with 64 vCPUs and 8 V100s) but no matter what zone I try I am not able to start the VM. Does anyone know if the other cloud providers have any availability? Thank you.
-David Tweedle

**Comment by davidtweedle** [comment]
Hi,
I am finding an available set up on another platform. But it is not exactly the same as the ones on Google Cloud Platform.
It has 128 vCPUs with 258GB RAM with 8 V100s.  Will this be acceptable to test my submission? Thank you.
-David Tweedle

**Reply by priyakasimbeg** [reply]
Hi David! Apologies for the delay in response. 
I just answered your question on Discord but I'll also copy my response here. 

Myself and my colleagues also had some availability issues on GCP the last week. We were able to get the GPU resources we need for our VMs after a couple of tries. Using a different cloud platform or setup is fine to develop your submission. If your setup has more  vCPUs (or more total RAM) I would just try to monitor your RAM usage while running your submission to make sure your submission will not run out of memory on our scoring hardware.
